### PR TITLE
 scripts: tweak release note script to further reduce manual effort

### PR DESCRIPTION
**Pull request by jseldess**
- Add a flag to exclude the list of per-contributor PRs.
- Add the email sign-up and Downloads section.
- Add a flag to exclude email sign-up and downloads section.
- Add the Doc Updates section.
- Move the Contributors section to the bottom and reformat
  the intro to be consistent with past release notes.
- Catch a few more release note category misspellings.

### CODE REVIEW

File: scripts/release-notes.py [hunk 1]
```diff
@@ -367,30 +371,30 @@ def spin():
...
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-alpha.20180122.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
```
Comment (knz) [inline scripts/release-notes.py]: 1) You can auto-generate the download links by using the `--until` argument perhaps?
2) I would recommend making the download section conditional on a command-line argument too.

**Review by knz — COMMENTED** [review]
Seems good!

File: scripts/release-notes.py (hunk 1 above)
Comment (jseldess) [reply scripts/release-notes.py]: Done. PTAL.

**Comment by jseldess** [comment]
@knz, please take another look.

**Review by knz — APPROVED** [review]
Thank you!

**Pull request merged**